backend/apps/matching/views/participant_info_view.py

```diff
@@ -20,6 +21,11 @@ class ParticipantInfoViewView(UpdateView):
     slug_url_kwarg = "uuid"
     slug_field = "uuid"
 
+    def dispatch(self, *args, **kwargs):
+        if self.request.user.participant().info.uuid not in self.request.path:
```

Comment:
In my understanding, hospitals should also be able to view it or we should at least have a toggle for this in the configurations

Reply:
yes, this should definitely be configurable, after all, it would be pretty useless for a hospital if they could not view the complete info of a participant